sbom/PhylumExt.toml

```
@@ -0,0 +1,6 @@
+name = "sbom"
+description = "NPM package manager hooks"
+entry_point = "main.ts"
+
+[permissions]
+read = ["package-lock.json", "requirements.txt", "poetry.lock", "gemfile.lock"]
```

For other extensions that take a filename as an argument, we've just given read access to the current working directory (for example, [here](https://github.com/phylum-dev/cli/blob/e6114475024c3c046b2684708d15a2a1e30c6f26/extensions/duplicates/PhylumExt.toml#L4))

```suggestion
read = ["./"]
```

---

Good suggestion!